deployment/persistent_chains/testnet-2.0/testnet-2.0-add-validator.md

```diff
@@ -1,154 +1,179 @@
+Check whether `catching_up` field is set to `false` after a while using the following query:
 
-    - address and pubkey can be found using 
-        ```bash
-        dcld keys show --output text "<admin-account-name>"
-        ```
+```bash
+dcld status
+```
 
-    3.4. Wait until your NodeAdmin key is approved by the quorum of Testnet 2.0 trustees
+It may take couple of minutes to catch up using `state-sync` depending on how far the `statesync snapshot` was from the current state of the network
 
-4. ### Run Validator Node
-    4.2. Set the following `state sync` parameters under `[statesync]` section in `$HOME/.dcl/config/config.toml` <br>
-    ```bash
-    [statesync]
-    enable = true
+### 4.7 Make the node a validator
 
-    rpc_servers = "https://on.test-net.dcl.csa-iot.org:26657,https://on.test-net.dcl.csa-iot.org:26657"
+```bash
+dcld tx validator add-node --pubkey="<protobuf JSON encoded validator-pubkey>" --moniker="<node-name>" --from="<admin-account-name>"
+```
 
-    trust_height = <trust-height>
-    trust_hash = "<trust-hash>"
-    ```
+> **_Note:_** Get `<protobuf JSON encoded validator-pubkey>` using `dcld tendermint show-validator` command
 
-    You can get `<trust-height>` and `<trust-hash>` parameters using the following command:
-    ```bash
-    curl -s https://on.test-net.dcl.csa-iot.org:26657/commit | jq "{height: .result.signed_header.header.height, hash: .result.signed_header.commit.block_id.hash}"
-    ```
+(once transaction is successfully written you should see "code": 0 in the JSON output.)
 
-    4.3. Set `seeds` parameter under `[p2p]` section in `$HOME/.dcl/config/config.toml`
-    ```bash
-    [p2p]
-    seeds = "8190bf7a220892165727896ddac6e71e735babe5@100.25.175.140:26656"
-    ```
-    Your validator node will use `Seed Node` to discover peers with `state sync` snapshots
+## 5 Validator Node Deployment Verification
 
-    4.4. Get the latest version of `persistent_peers.txt` file, containing all other validator peer addresses, from the community and put it to the same directory where `run_dcl_node` script is.
-    ```bash
-    touch persistent_peers.txt
-    ```
+### 5.1 Check the account presence on the ledger: `dcld query auth account --address="<address>"`
 
-    4.5. Run Validator Node
-    ```bash
-    chmod u+x ./run_dcl_node
-    ./run_dcl_node -c "testnet-2.0" "<node-name>"
-    ```
-
-    4.6. Verify node is running <br>
-    - Execute `source $HOME/.profile` to take the updated `$PATH` into effect, now
-    it includes the directory containing the current version of `dcld` binary (if
-    you have not modified or commented out the line doing the corresponding
-    `$PATH` assignment in `run_dcl_node` script):
-        ```bash
-        source $HOME/.profile
-        ```
+### 5.2 Check the node service is running: `systemctl status dcld`
 
+### 5.3 Check the node gets new blocks: `dcld status`. Make sure that `result.sync_info.latest_block_height` is increasing over the time (once in about 5 sec)
```

Comment:
These headers seem a bit weird. Probably move the commands (the text after `:`) into the body?